lrauv_ignition_plugins/src/TethysCommPlugin.cc

```diff
@@ -173,6 +173,13 @@ void ROSToFSK(ignition::math::Vector3d &_vec)
   _vec.Z(-_vec.Z());
 }
 
+// Convert a pose in ENU to NED.
+ignition::math::Pose3d ENUToNED(ignition::math::Pose3d &_enu)
```

Comment:
Minor knit: document argument. Argument can also be marked `const`.

Reply:
Good call 92dcef0894521140d33c562458ea86c710173b13